ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h

```diff
@@ -64,21 +77,31 @@ class LocalAssemblerData : public ProcessLib::LocalAssemblerInterface<GlobalMatr
         assert(local_matrix_size == ShapeFunction::NPOINTS * NUM_NODAL_DOF);
     }
 
-    void assemble(double const /*t*/, std::vector<double> const& /*local_x*/) override
+    void assemble(double const /*t*/, std::vector<double> const& local_x) override
     {
         _localA.setZero();
         _localRhs.setZero();
 
         IntegrationMethod integration_method(_integration_order);
         unsigned const n_integration_points = integration_method.getNPoints();
 
-        for (std::size_t ip(0); ip < n_integration_points; ip++) {
+        for (std::size_t ip(0); ip < n_integration_points; ip++)
+        {
             auto const& sm = _shape_matrices[ip];
             auto const& wp = integration_method.getWeightedPoint(ip);
-
             auto const k = _process_data.hydraulic_conductivity(_element);
+
             _localA.noalias() += sm.dNdx.transpose() * k * sm.dNdx *
                                  sm.detJ * wp.getWeight();
+
+            // Darcy velocity only computed for output.
+            auto const darcy_velocity = (k * sm.dNdx *
```

Comment:
The velocity is not only computed for output, it also has to be computed for the governing equations for heat or mass transport. Therefore, the computation can not be in postTimestep().